static/sass/styles.scss

```diff
@@ -1,6 +1,11 @@
 // vanilla patterns
 @import 'vanilla-framework/scss/vanilla';
-@include vanilla-base;
+
+$color-brand: #00302f; // TODO: this is a temprary brand colour
+$color-accent: $color-brand; // temporary for strip background
+$color-accent-background: $color-accent;
```

Comment:
I believe this is default styling: https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_settings_colors.scss#L23

Reply:
Yes it is. It didn't work for me because I defined values after importing vanilla.

Moving assign before import fixes this and makes it possible to assign `$color-brand` only and leave `$color-accent-background` as default.